tests/Install-Scripts.Test/GivenThatIWantToInstallDotnetFromAScript.cs

```diff
@@ -210,8 +215,7 @@ public void WhenInstallingTheSdk(string channel, string? quality, string version
         [MemberData(nameof(InstallRuntimeFromChannelTestCases))]
         public void WhenInstallingDotnetRuntime(string channel, string? quality, string versionRegex)
         {
-            if (channel == "release/5.0" ||
-                channel == "5.0" && !RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            if (channel == "release/5.0")
```

Comment:
should we replace these cases with quality?

Reply:
They should still continue to work for .NET < 6.0 without Quality option.

Reply:
I tried also adding 5.0 tests with quality option, but again, the resources are not available in the storage. Since 5.0 is already out, we probably won't have them. But I will check.